compiler/rustc_mir_transform/src/lib.rs

```diff
@@ -477,10 +481,13 @@ pub fn run_analysis_to_runtime_passes<'tcx>(tcx: TyCtxt<'tcx>, body: &mut Body<'
         pm::run_passes(
             tcx,
             body,
-            &[&remove_uninit_drops::RemoveUninitDrops, &simplify::SimplifyCfg::RemoveFalseEdges],
+            &[
+                &remove_uninit_drops::RemoveUninitDrops,
+                &simplify::SimplifyCfg::RemoveFalseEdges,
+                &Lint(post_drop_elaboration::CheckLiveDrops),
+            ],
             None,
         );
-        check_consts::post_drop_elaboration::check_live_drops(tcx, body); // FIXME: make this a MIR lint
```

Comment:
`pm::run_passes` has a bit of extra logic in it for running MIR linting/validation as well as `-Zmir-enable-passes`. So this isn't necessarily a no-op, but I'm down with following the FIXME.

Reply:
I think this is just a comment, not a request for a change, right?